model-setup-examples/basic-elevon.md

```diff
@@ -0,0 +1,22 @@
+## Basic Elevon
+
+Elevons are control surfaces that combine the functions of the elevator and the aileron.
```

Comment:
Yes, I think so. I am flying quad and helicopters. I would love something that describes the type of plane this setup is for.